app/views/layouts/application.mobile.haml

```diff
@@ -3,56 +3,52 @@
 -# the COPYRIGHT file.
 
 !!!
-%html{:lang => I18n.locale.to_s, :dir => (rtl?) ? 'rtl' : 'ltr'}
-  %head
+%html{:lang => I18n.locale.to_s, :dir => (rtl?) ? "rtl" : "ltr"}
```

Comment:
- Use the new Ruby 1.9 hash syntax.
- Don't use parentheses around a method call.